miro_preprocessor/miro_copy_s3_asset/variables.tf

```diff
@@ -1,13 +1,23 @@
-variable "bucket_miro_images_sync_arn" {}
+variable "bucket_source_asset_arn" {}
 
 variable "lambda_error_alarm_arn" {}
 
 variable "topic_miro_copy_s3_asset_arn" {}
 
-variable "bucket_miro_images_public_arn" {}
+variable "bucket_destination_asset_arn" {}
 
-variable "topic_miro_image_to_dynamo_arn" {}
+variable "topic_forward_sns_message_arn" {
+  default = ""
```

Comment:
remove this?

Reply:
It's used by the lambda that copies the derivative, but not by the one that copies the master. That's why it's there but empty by default

Reply:
Maybe add that into the description?

Reply:
Yep, makes sense